### PR TITLE
intensities by individauls

### PR DESCRIPTION
**Pull request by jhsiao999**

### CODE REVIEW

**Comment by jhsiao999** [comment]
@jdblischak I looked at the conflict. And below is the updated index.Rmd that you can use to resolve conflicts. I also made additional changes to the index.Rmd. I am leaving it to you to do `Resolve Conflicts` and `Commit Merge`. Thanks!

```
---
title: "Home"
output:
  html_document:
    toc: false
---

Website to share results of fucci-seq project.


## Image analysis

* [CONFESS preliminary results](confess-prelim.html)
* [Processing images - from images to intensities](images-process.html)
* [Initial analysis](images-qc.html)
* [Follow-up using alternative metrics for signals](images-qc-followup.html)
* [Explore measurements variation by individual labels](images-qc-labels.html)
* [Inspect images for multi-nuclei samples](images-multiple-nuclei.html)

## QC

* [Sequencing depth](totals.html)
* [Reads versus molecules](reads-v-molecules.html)
```

File: analysis/_site.yml [hunk 1]
```diff
@@ -1,25 +1,28 @@
 name: fucci-seq
-output_dir: "../docs"
-include: [".nojekyll"]
-exclude: ["include"]
+output_dir: ../docs
+include: .nojekyll
+exclude: include
 navbar:
```
Comment (jdblischak) [inline analysis/_site.yml:5]: Why did you change this? Was it causing problems? Did you confirm it still works as expected?

File: analysis/_site.yml [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
-    toc_float: true
+    toc: yes
+    toc_float: yes
     theme: cosmo
     highlight: textmate
     include:
       before_body: include/header.html
       after_body: include/footer.html
+    lib_dir: site_libs
+    self_contained: no
```
Comment (jdblischak) [inline analysis/_site.yml:27]: Why did you add these manually? These should be automatically added by `rmarkdown::render_site()`.

**Comment by jdblischak** [comment]
Are you having trouble with workflowr? I noticed you committed the R Markdown and HTML in the same commit.

File: analysis/_site.yml (hunk 1 above)
Comment (jhsiao999) [reply analysis/_site.yml:5]: I previewed index.Rmd and it worked as expected. I didn't change these. After I finished knitting the two Rmds that I want to commit, I saw these files changed when I do `git status`... Not sure at which point of the process that these were generated.

File: analysis/_site.yml (hunk 2 above)
Comment (jhsiao999) [reply analysis/_site.yml:27]: I also didn't add these manually... I didn't make changes to`_site.yml`.

**Comment by jhsiao999** [comment]
No I am not. I had closed my Rstudio session already and was working on the terminal. So I decided to do a quick update on my fucci-seq repo. Perhaps these problems come from my `quick update` on the terminal...

File: analysis/_site.yml (hunk 1 above)
Comment (jdblischak) [reply analysis/_site.yml:5]: What version of rmarkdown are you running? `packageVersion("rmarkdown")` Maybe editing `_site.yml` is a newer feature?

**Comment by jdblischak** [comment]
RStudio is great for interactively developing the analysis, but in the future please try to take advantage of the reproducibility features we have available. Whenever I am finished with an analysis and am ready to commit the changes, here is what I do:

```
# Activate conda environment
source activate fucci-seq
# Start R
R
# Run wflow_publish
> library("workflowr")
> wflow_publish(c("analysis/index.Rmd", "analysis/newfile.Rmd"), "Add new analysis")
```

File: analysis/_site.yml (hunk 1 above)
Comment (jhsiao999) [reply analysis/_site.yml:5]: it's 1.8

**Comment by jhsiao999** [comment]
OK. Will try next time. Old habits die hard...

File: analysis/_site.yml (hunk 1 above)
Comment (jdblischak) [reply analysis/_site.yml:5]: Strange. I didn't see anything in the [NEWS file](https://github.com/rstudio/rmarkdown/blob/master/inst/NEWS) to indicate this change. Not sure what caused this.
